docs/the-new-architecture/use-app-template.md

```diff
@@ -3,5 +3,61 @@ id: use-app-template
 title: Using the New App Template
 ---
 
-This section describes the new app template generated by the React Native CLI and how to use it properly.
-It should explain how to create a new app, which command should be issued to install the dependencies and to run it on the different platforms.
+This page will help you create a new React Native app that uses the new architecture.
+
+## Installing dependencies
+
+Start by setting up your development environment as described in [Getting Started](getting-started.md), under "React Native CLI Quickstart". Stop when you reach "React Native Command Line Interface", and resume here.
```

Comment:
Yeah, unfortunately docusaurus does not support anchor links here. This is originally why I started thinking of ways to combine "Getting Started" with this page.

Reply:
cc @Simek I've noticed this behavior as well in the website (anchor links on tabbed pages are sometimes broken). Do you know if someone looked into this in the past?

Reply:
There's a potential way around this that I found, but I'm not sure if we want it.

If we uniquely title the headers per tab, anchor links to headers will work. The problem is that multiple combinations of tabs have "Installing dependencies" as a header.

We could instead repeat the tab values in the headers, manually, which creates a unique anchor link:

<img width="1461" alt="Screen Shot 2022-04-07 at 3 59 55 PM" src="https://user-images.githubusercontent.com/5252268/162333018-94cca1d4-a054-4449-9de2-0a622e7bf8e2.png">

Obviously that's more verbose. But I think it could work, because:

* The page is somewhat long and when I'm doing a multi-platform setup in a guide like this where I might be doing multiple things at once, it's honestly helpful to be able to have a reminder of which options I'm looking at at the moment without having to scroll up
* We could only do it on main headers, or as needed for linking

But still, less ideal than having a docusaurus fix for this.

Reply:
It's a known issue, that hash links do not work correctly in Tabs or partials (for tabs they only work if you have correct tabs stored in local storage). Not sure if we can do anything, other that stop using tabs and refactor the content. But we need to keep in mind that part of docs is quite crucial and we need to carefully think about changes to this page.

Refs:
* https://github.com/facebook/docusaurus/issues/7008
* https://docusaurus.canny.io/feature-requests/p/link-to-tab